apt.txt

```diff
@@ -1,4 +1,7 @@
+git
```

Comment:
No worries, already removed. That _was_ annoying when `apt-get install git` didn't work and you had to train yourself to type `git-core`. I'm glad they "fixed" it, but it's still annoying. Like on Ubuntu, there's already a dock applet nobody uses called `docker`, so you have to install `docker.io` from the repo. Of course I use the official repo, that's `lxc-docker`. This is why we can't have nice things.